crates/hyperqueue/src/client/output/cli.rs

```diff
@@ -1044,6 +1045,99 @@ impl Output for CliOutput {
             job_id.to_string().color(colored::Color::Green),
         );
     }
+
+    fn print_explanation(
+        &self,
+        task_id: TaskId,
+        worker_id: WorkerId,
+        explanation: &TaskExplanation,
+    ) {
+        let all = explanation.variants.len();
+        let count: usize = explanation
```

Comment:
`count` -> `runnable_variants` (it was hard for me to understand how this works)